internal/workload/manager.go

```diff
@@ -238,6 +268,11 @@ func (w *WorkloadManager) ensureWorkloadsFromManifestsAreRunning() error {
 				// Workload is not running - start
 				err = w.workloads.Start(&podWithPath.pod)
 				if err != nil {
+					w.eventsQueue = append(w.eventsQueue, &models.EventInfo{
+						Message: err.Error(),
+						Reason:  "Started",
```

Comment:
Shouldn't it be `NotStarted`? IMO `Started` would describe event of successful start of a pod.

Reply:
Right. Actually k8s uses `Failed`, so I think I will reuse it.